Makefile

```diff
@@ -85,13 +99,20 @@ build: # Build REANA client and cluster components.
 	reana-dev install-client && \
 	reana-dev install-cluster && \
 	reana-dev git-submodule --update && \
-	reana-dev docker-build -t latest && \
-	reana-dev git-submodule --delete
+	reana-dev docker-build -b DEBUG=${DEBUG} && \
+	reana-dev git-submodule --delete && \
+	if [ "${DEBUG}" -gt 0 ]; then \
+		echo ${ccyellow} && \
+		echo "Please run minikube mount in a new terminal to have live code updates." && \
+		echo "For more information visit the documentation: \nhttps://reana-cluster.readthedocs.io/en/latest/developerguide.html#deploying-latest-master-branch-versions" && \
+		echo ${ccend}; \
+	fi
 
 deploy: # Deploy/redeploy previously built REANA cluster.
+	if [ "${DEBUG}" -eq 0 ] || ([ "${DEBUG}" -gt 0 ] && [ $$(ps -ef | grep -i '[m]inikube mount' | echo "$$?") -eq 0 ]); then \
 	source ${HOME}/.virtualenvs/${VENV_NAME}/bin/activate && \
 	eval $$(minikube docker-env --profile ${MINIKUBE_PROFILE}) && \
-	reana-cluster -f ${PWD}/../reana-cluster/reana_cluster/configurations/reana-cluster-latest.yaml down && \
+	reana-cluster -f ${PWD}/../reana-cluster/reana_cluster/configurations/reana-cluster-minikube$(addprefix -, ${CLUSTER_CONFIG}).yaml down && \
```

Comment:
I think you did not push latest changes to `REANA-Cluster` regarding `reana-cluster-minikube.yaml` naming... The files were removed there:

```console
$ git log master..upstream/pr/199 --stat | grep configurations
 .../configurations/reana-cluster-dev.yaml          | 57 ----------------------
 .../configurations/reana-cluster-latest.yaml       | 27 ----------
 reana_cluster/configurations/reana-cluster.yaml    | 35 -------------
```